typescript/ecs/ecs-service-with-logging/index.ts

```diff
@@ -0,0 +1,42 @@
+import ec2 = require('@aws-cdk/aws-ec2');
+import { InstanceType } from '@aws-cdk/aws-ec2';
+import ecs = require('@aws-cdk/aws-ecs');
+import cdk = require('@aws-cdk/cdk');
+
+class WillkommenECS extends cdk.Stack {
+  constructor(scope: cdk.App, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    const vpc = new ec2.VpcNetwork(this, 'MyVpc', { maxAZs: 2 });
+    const cluster = new ecs.Cluster(this, 'Ec2Cluster', { vpc });
+    cluster.addDefaultAutoScalingGroupCapacity({
```

Comment:
This is deprecated -- should be `addCapacity`. Also, could you please update your package.json to the latest (0.25.3)?